cypress/e2e/agreements/agreement-line-create-multiple.cy.js

```diff
@@ -0,0 +1,234 @@
+import { MultiColumnListCell } from '@folio/stripes-testing';
+import { getRandomPostfix } from '../../support/utils/stringTools';
+import DateTools from '../../support/utils/dateTools';
+
+import AgreementAppInteractor from '../../support/fragments/agreements/AppInteractor';
+import AgreementViewInteractor from '../../support/fragments/agreements/AgreementViewInteractor';
+import LocalKBAdminAppInteractor from '../../support/fragments/local-kb-admin/AppInteractor';
+import AgreementLineFormInteractor from '../../support/fragments/agreements/AgreementLineFormInteractor';
+import PackageViewInteractor from '../../support/fragments/agreements/PackageViewInteractor';
+import AgreementLineViewInteractor from '../../support/fragments/agreements/AgreementLineViewInteractor';
+
+// file - package - agreement
+const fileName = 'simple_package_for_updates_1.json';
+const packageName = 'Simple package to test updating package metadata';
+const agreementName = `Agreement test ${getRandomPostfix()}`;
+const description = `Agreement line test description ${getRandomPostfix()}`;
+
+// agreement
+const agreement = {
+  name: agreementName,
+  status: 'Active',
+  startDate: DateTools.getCurrentDate(),
+};
+
+// users
+const editUser = {
+  username: `editTest${getRandomPostfix()}`,
+  password: 'editTest',
+};
+
+const editPermissions = [
+  'ui-agreements.agreements.edit',
+  'ui-agreements.resources.edit',
+  'ui-local-kb-admin.jobs.edit',
+];
+
+const viewUser = {
+  username: `viewTest${getRandomPostfix()}`,
+  password: 'viewTest',
+};
+
+const viewPermissions = [
+  'ui-agreements.agreements.view',
+  'ui-agreements.resources.view',
+];
+
+let isChecked = true;
+
+function deleteFirstAgreementLine() {
+  AgreementViewInteractor.paneExists(agreementName);
+  AgreementViewInteractor.openFirstAgreementLine();
+  AgreementLineViewInteractor.paneExists('pane-view-agreement-line');
+  AgreementLineViewInteractor.delete('pane-view-agreement-line');
+  AgreementViewInteractor.paneExists(agreementName);
+}
+
+describe('Agreement line test', () => {
+  before('before hook', () => {
+    console.log('createUser');
+    cy.createUserWithPwAndPerms(editUser, editPermissions);
+    cy.createUserWithPwAndPerms(viewUser, viewPermissions);
+
+    console.log('do not hide internal agreements knowledgebase');
+    cy.getAdminToken();
+    cy.getAgreementsGeneralSettings().then((settings) => {
+      if (settings?.hideEResourcesFunctionality === true) {
+        isChecked = false;
+        cy.setAgreementsGeneralSettings({ hideEResourcesFunctionality: false });
+      }
+    });
+    cy.login(Cypress.env('login_username'), Cypress.env('login_password'));
+    cy.getAdminToken();
+    LocalKBAdminAppInteractor.openLocalKbAdminApp();
+    LocalKBAdminAppInteractor.uploadJsonFileAndAwaitCompletion(fileName);
```

Comment:
It would be better to check first if that package is already uploaded (in the packages section of the local kb search)